### DISCUSSION TITLE
有使用定制翻译的方法吗？

### DISCUSSION THREAD

**Discussion by gyeolls** [chinosk6/gakuen-imas-localify#14] [Q&A]
我想在韩国制作韩语翻译，所以想使用chinosk6的相应程序。 更换了字体，修改了/app/src/main/assets/GakumasTranslationData/local-files/resource/内的txt文件，但仍以中文打印。 为了以防万一，修改了其他路径名称相同的csv文件，即使构建程序，也会同样输出中文。
除了该文件之外，还有什么需要修改的吗？

 另外，为了将从acb、awb文件中提取的hca文件转换成wav文件，

CriAtomExACb.Decrypt(ulong, ulong)
CriAtomPlugin.DecryptAcb(IntPtr, ulong, ulong)的值,但我亲自修改了hook.cpp,发现apk无法构建。 这个也可以请求帮助吗？

 因为我不会说中文，所以用了翻译器。 如果不能理解的话，很抱歉。

**Comment by darwintree** [comment]
The txt files are communication scripts. Currently other tracked strings are also in generic.json and localization.json.
Besides, to replace the extracted resources, you need to change the version.txt every time

**Reply by gyeolls** [reply]
Thank you. It's been solved thanks to you. Do you happen to know how to get the hca key as well?

**Reply by darwintree** [reply]
> Thank you. It's been solved thanks to you. Do you happen to know how to get the hca key as well?

I've tried reading the game resource `.acb` audio files and I think no extra decryption is needed. As for other files, I'm sorry I don't know any more

**Comment by Badmingrams** [comment]
We are now working on source string sharing and wonder if the korea community is willing to join.
Discord link in the issue ⬇️
https://github.com/NatsumeLS/Gakumas-Translation-Data-EN/issues/1

**Reply by Badmingrams** [reply]
@gyeolls

**Reply by gyeolls** [reply]
Of course there is. I'm going to go into Discord
